internal/service/kinesisanalyticsv2/application_test.go

```diff
@@ -1272,6 +1272,51 @@ func TestAccKinesisAnalyticsV2Application_FlinkApplication_restoreFromSnapshot(t
 	})
 }
 
+func TestAccKinesisAnalyticsV2Application_FlinkApplication_snapshotToggleRestoreTypeBug(t *testing.T) {
```

Comment:
`Bug` suffix does not provide information what to expect, maybe `snapshotToggleWhenSkipRestoreFromSnapshot`? Test itself looks good